docs/verifiable-payid.md

```diff
@@ -0,0 +1,77 @@
+6. Sign the PayID with the user identity key to create a verified PayID and address payload: `sign`.
+7. Remove the unverified crypto address: `crypto-address remove rDk7FQvkQxQQNGTtfM2Fr66s7Nm3k87vdS`. The output payload should have this format:
+```js
+{
+  "payId": "verifiabledemo$127.0.0.1",
+  "addresses": [],
+  "verifiedAddresses": [
+    {
+      "payload": "{\"payId\":\"test$example.com\",\"payIdAddress\":{\"paymentNetwork\":\"XRPL\",\"environment\":\"TESTNET\",\"addressDetailsType\":\"CryptoAddressDetails\",\"addressDetails\":{\"address\":\"rDk7FQvkQxQQNGTtfM2Fr66s7Nm3k87vdS\"}}}",
+      "signatures": [
+        {
+          "protected": "eyJuYW1lIjoiaWRlbnRpdHlLZXkiLCJhbGciOiJFUzI1NksiLCJ0eXAiOiJKT1NFK0pTT04iLCJiNjQiOmZhbHNlLCJjcml0IjpbImI2NCIsIm5hbWUiXSwiandrIjp7ImNydiI6InNlY3AyNTZrMSIsIngiOiI2S0dtcEF6WUhWUm9qVmU5UEpfWTVyZHltQ21kTy1xaVRHem1Edl9waUlvIiwieSI6ImhxS3Vnc1g3Vjk3eFRNLThCMTBONUQxcW44MUZWMjItM1p0TURXaXZfSnciLCJrdHkiOiJFQyIsImtpZCI6Im4zNlhTc0M1TjRnNUtCVzRBWXJ5d1ZtRE1kUWNEV1BJX0RfNUR1UlNhNDAifX0",
+          "signature": "rsoKeVLzwg2PpGRK0S10fpmh5WNtttF9dyJgSv3USEr4aN3bUBzp5ImRQo8wlh3E00GtZ2cse-lhoQ4zJKj0Jw"
+        }
+      ]
+    }
+  ]
+}
+```
+8. Upload the verified PayID. If you are using the PayID Admin API version from 2020-08-25 or later, POST the payload from Step 7 to the Admin API endpoint. See [PayID API Reference](https://api.payid.org). As mentioned, you must have a PayID server deployed to use this API.
+    If you use  the PayID Admin API version from before 2020-08-25, then you must modify this payload.
+    * `identityKey` should have the contents of the `protected` field in Step 7.
+    * `verifiedAddresses` should be the un-stringified JSON contents of the `payload` field in Step 7.
+    * `identityKeySignature` should be the `signature` field in Step 7.
+   The payload for older versions of the PayID Admin API appears as follows:
+```js
+{
+   "payId": "verifiabledemo$127.0.0.1",
+   "identityKey": "eyJuYW1lIjoiaWRlbnRpdHlLZXkiLCJhbGciOiJFUzI1NksiLCJ0eXAiOiJKT1NFK0pTT04iLCJiNjQiOmZhbHNlLCJjcml0IjpbImI2NCIsIm5hbWUiXSwiandrIjp7ImNydiI6InNlY3AyNTZrMSIsIngiOiI2S0dtcEF6WUhWUm9qVmU5UEpfWTVyZHltQ21kTy1xaVRHem1Edl9waUlvIiwieSI6ImhxS3Vnc1g3Vjk3eFRNLThCMTBONUQxcW44MUZWMjItM1p0TURXaXZfSnciLCJrdHkiOiJFQyIsImtpZCI6Im4zNlhTc0M1TjRnNUtCVzRBWXJ5d1ZtRE1kUWNEV1BJX0RfNUR1UlNhNDAifX0",
+   "addresses": [],
+   "verifiedAddresses": [
+       {
+           "paymentNetwork": "XRPL",
+           "environment": "TESTNET",
+           "details": {
+               "address": "rDk7FQvkQxQQNGTtfM2Fr66s7Nm3k87vdS"
+           },
+           "identityKeySignature": "rsoKeVLzwg2PpGRK0S10fpmh5WNtttF9dyJgSv3USEr4aN3bUBzp5ImRQo8wlh3E00GtZ2cse-lhoQ4zJKj0Jw"
+       }
+   ]
+}
+```
+9. Retrieve and verify the verified PayID with the PayID CLI payid-utils tool. These commands do not use interactive mode.
+    `payid-utils`
```

Comment:
payid-utils did not work for me from the command line. Am I missing something here? The other commands work.

Reply:
`payid-utils` is no longer the CLI tool: https://github.com/payid-org/payid-cli

Look at that page for the updated commands.